dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapper.java

```diff
@@ -144,7 +149,10 @@ public EventOperationParams map(RequestParams requestParams) throws BadRequestEx
             .attributeFilters(attributeFilters)
             .events(UID.toValueSet(eventUids))
             .enrollments(UID.toValueSet(requestParams.getEnrollments()))
-            .includeDeleted(requestParams.isIncludeDeleted());
+            .includeDeleted(requestParams.isIncludeDeleted())
+            .includeRelationships(
```

Comment:
I think we should follow the same strategy as the other entities and have `EventParams` in `EventOperationParams` instead of `includeRelationships`.
`EventParams` is already correctly populated in `EventFieldsParamMapper`, we can use it here instead of using directly `FieldFilterService`